client/components/shared/styles.js

```diff
@@ -0,0 +1,138 @@
+import styled, { css } from 'styled-components';
+
+// -- TEXT ELEMENTS -- //
+
+// HEADINGS //
+export const HugeText = styled.h2`
+  font-family: Roboto;
```

Comment:
H1 - H6 already has roboto as font-family :)

Reply:
I just copied straight from Christin's Figma for each font.

Reply:
I removed the 'Roboto' on H's and made all paragraphs 'Open Sans' in the general styles.